smallfile_wrapper/ci_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+source ci/common.sh
+
+# Build image for ci
+podman build --tag=quay.io/cloud-bulldozer/smallfile:snafu_ci -f smallfile_wrapper/Dockerfile . && podman push quay.io/cloud-bulldozer/smallfile:snafu_ci
+
+cd ripsaw
+
+sed -i 's/smallfile:master/smallfile:snafu_ci/g' roles/smallfile-bench/tempates/*
+
+# Build new ripsaw image
+update_operator_image snafu_ci
+
+sed -i '/trap finish EXIT/d' tests/test_smallfile.sh
+
+rm -f uuid
+(
+source tests/test_smallfile.sh || :
+
+# Get UUID
+uuid=`kubectl -n my-ripsaw get benchmarks -o jsonpath='{.items[0].status.uuid}'`
+
+finish
+echo $uuid > uuid
+)
+uuid=`cat uuid`
+
+cd ..
+
+# Define ES server info (this is the same from the test cr
+es_server="marquez.perf.lab.eng.rdu2.redhat.com"
```

Comment:
why do es_server and es_port get defined inside wrappers instead of a shared environment variable or file?

Reply:
they now get defined in common.sh which is sourced in.